src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -2788,6 +2790,26 @@ class spell_pri_vampiric_touch : public AuraScript
     }
 };
 
+// 390615 - From Darkness Comes Light (Talent)
+class spell_pri_from_darkness_comes_light : public AuraScript
+{
+    bool CheckEffectProc(AuraEffect const* /*aurEff*/, ProcEventInfo& eventInfo)
+    {
+        return eventInfo.GetSpellInfo()->Id == SPELL_PRIEST_PURGE_THE_WICKED || eventInfo.GetSpellInfo()->Id == SPELL_PRIEST_PURGE_THE_WICKED_PERIODIC || eventInfo.GetSpellInfo()->Id == SPELL_PRIEST_SHADOW_WORD_PAIN;
```

Comment:
Why are you checking spellId if you added spellFamilyMasks in spell_proc for the same spell?

Reply:
agree makes sense